### ISSUE TITLE
Make a separate "Teams" page

### ISSUE THREAD

**Issue by fricklerhandwerk** [NixOS/nixos-homepage#1511]
**Problem**
The [community page](https://nixos.org/community/) is currently about too many things:
- communication channels
- NixCon
- meetups
- RFC process
- teams
- foundation

**Proposal**

Split out the Teams part.

Ideally also move the foundation stuff on a separate page linked from the footer.
The RFC process diagram could go into the steering committee page.

I'd just do it myself, but the website has so much design stuff that one can't just move a bunch of text to a different place, and I can't make the time to figure out what to do until it looks correct. (Side note: It would be great if the contribution process would boil down to moving text around.)

**Additional context**
There were two notable discussions on the role of teams in 2023:
- https://discourse.nixos.org/t/what-do-you-think-of-teamification/34849
- https://discourse.nixos.org/t/nixcon-governance-workshop/32705/7

My takeaway is that we need to help people doing the technical work to get more visibility and access to those interested in helping out, and those interested in helping out a way to find people doing the technical work.

Yesterday I discussed with @imincik that we should emphasize the existence of active groups working on particular things, and  and that joining one is a fast track to making change happen without getting frustrated. A separate navigation item would be a first step in that direction.

- [ ] I already created a Pull Request

**Comment by piegamesde** [comment]
Yes, please. 95% of the time where I go to the home page is to check on who is currently in a certain team or how to contact them. Having to scroll down through a lot of community stuff which doesn't interest me is kinda bothersome.
